### PR TITLE
uuid cleanup

### PR DESCRIPTION
**Pull request by cjkeilig**
Indenting errors are fixed and variable names fixed, I reforked and re-cloned and then put my changes back in.

### CODE REVIEW

**Comment by djw8605** [comment]
Why did you make this?  Just keep adding to the previous pull request.

**Comment by cjkeilig** [comment]
I was having a git nightmare so I reset my fork, the old request shows "from 'unknown repository'" so I thought I'd make a new one.

**Comment by djw8605** [comment]
Yeah, resetting the fork would have screwed things up.

Ok, I'll take a look at this pull request.

File: HCCGo/app/js/PreferencesManager.js [hunk 1]
```diff
@@ -2,15 +2,17 @@
...
-preferencesModule.factory('preferencesManager',['$log', '$q', function($log, $q) {
+preferencesModule.factory('preferencesManager',['$log', '$q', 'filePathService', function($log, $q, filePathService) {
   
   var clustersDefer;
-  
+  var preferencesDefer;
+  var preferencePath = filePathService.getPreferencePath();
+  var fs = require("fs");
+
   var init = function() {
```
Comment (djw8605) [inline HCCGo/app/js/PreferencesManager.js]: Can you add new jsdoc comments for this `init` function.  Like below.

**Pull request merged**